test/SpokePoolClient.v3Events.ts

```diff
@@ -362,7 +373,10 @@ describe("SpokePoolClient: Event Filtering", function () {
         exclusiveRelayer,
       } as DepositWithBlock);
       await originSpokePoolClient.update(["V3FundsDeposited"]);
-      const deposit = originSpokePoolClient.getDeposits().at(-1);
+      let deposit = originSpokePoolClient.getDeposits().at(-1);
```

Comment:
This looks great. Do we have any tests where we mock "old" events (not V2, but V3 pre-bytes32 upgrade)? Would be nice to see spoke pool client query both in the same test and produce the same internal data structures via getDeposits/getFills

Reply:
Yeah, the branch pxrl/filledRelayMessageHash does this in its tests.